ccd-definition/CaseEventToFields/ClaimantResponseLRspec-CUI-nonprod.json

```diff
@@ -206,23 +206,35 @@
   {
     "CaseTypeID": "CIVIL",
     "CaseEventID": "CLAIMANT_RESPONSE_SPEC",
-    "CaseFieldID": "applicant1ProceedWithClaimSpec2v1",
+    "CaseFieldID": "applicantDefenceResponseDocument",
```

Comment:
Is it better to change a more meaningful name, because some of the Document field using xxxDocument. It will make me confuse that it is a document

Reply:
updated flag name.